server/text_generation_server/habana_quantization_env.py

```diff
@@ -5,7 +5,9 @@
 
```

Comment:
need to update readme
https://github.com/huggingface/tgi-gaudi/blob/0ca54b55f84e2fcb5b05bb887d736da1ebe76343/README.md?plain=1#L105